newspaper/extractors.py

```diff
@@ -195,6 +199,16 @@ def parse_date_str(date_str):
             if datetime_obj:
                 return datetime_obj
 
+        JSONLD_TAGS = [
+            'datePublished'
+        ]
+        jsonlde = JsonLdExtractor()
+        jsonlddata = jsonlde.extract_items(doc)
```

Comment:
Is this call expensive? I worry since we are running this over the entire html `doc`